assets/airports/cyow.json

```diff
@@ -0,0 +1,1279 @@
+                ["AVVON*"],
+                ["IKLAX*"],
+                ["KODEX*"],
+                ["LETAK*"],
+                ["OLIGO*"],
+                ["RADEN*"],
+                ["TUKIR*"]
+            ]
+        }
+    },
+    "stars": {
+        "CAPTL4": {
+            "icao": "CAPTL4",
+            "name": "Capital Four",
+            "entryPoints": {
+                "MIVOK": ["MIVOK"],
+                "ELSUB": ["ELSUB"]
+            },
+            "body": [["KANIK", "A230"]],
+            "rwy": {
+                "07": [["LANRK", "A100-"], ["VISOL", "A30"], "#071"],
+                "14": [["LANRK", "A160-"], ["VISOL", "A50"], ["SEBEG", "A50"], ["MEGNO", "A35"], "OMTUN", "#320"],
+                "25": [["LANRK", "A160-"], ["VISOL", "A50"], "ULAGU", ["BOREG", "A50|S230"], ["ADVOS", "A30|S210"], "KAREB", "#084"],
+                "32": [["LANRK", "A160-"], ["VISOL", "A50"], "ULAGU", ["DEMKO", "A50|S210"], ["VOLAG", "A30"], "DUKTI", "#140"],
+                "22": []
+            },
+            "draw": [
+                ["MIVOK*", "KANIK"],
+                ["ELSUB*", "KANIK", "LANRK", "VISOL"],
+                ["VISOL", "ULAGU", "BOREG", "ADVOS", "KAREB"],
+                ["ULAGU", "DEMKO", "VOLAG", "DUKTI"],
+                ["VISOL", "SEBEG", "MEGNO", "OMTUN"]
+            ]
+        },
+        "DEANS8": {
+            "icao": "DEANS8",
+            "name": "Deans Eight",
+            "entryPoints": {
+                "ART": ["ART"],
+                "BUGSY": ["BUGSY", ["SAVAL", "A50+"], ["TAPVO", "A180-|A40+"]]
+            },
+            "body": [],
+            "rwy": {
+                "07": [["CYRIL", "A120"], ["CAVUR", "A75|S230-"], ["AMAGI", "A50|S210-"], ["NOVUX", "A50"], ["TILOG", "A30"], "EBNAP", "#247"],
+                "14": [["CYRIL", "A120"], "CAVUR", "LITRI", ["APMUP", "A50|S210-"], "MERKO", ["ILOGI", "A35"], "PETLU", "#320"],
+                "25": [["CYRIL", "A120"], ["TEXEN", "A50|S230-"], "ADVOS", "KAREB", "#084"],
+                "32": [["CYRIL", "A80-"], ["TEXEN", "A30"], "#320"],
+                "22": []
+            },
+            "draw": [
+                ["ART*", "CYRIL"],
+                ["BUGSY*", "SAVAL", "TAPVO", "CYRIL"],
+                ["CYRIL", "TEXEN", "ADVOS"],
+                ["CYRIL", "CAVUR", "LITRI", "APMUP", "MERKO", "ILOGI", "PETLU"],
+                ["CAVUR", "AMAGI", "NOVUX", "TILOG", "EBNAP"]
+            ]
+        },
+        "LEAMY3": {
+            "icao": "LEAMY3",
+            "name": "Leamy Three",
+            "entryPoints": {
+                "BEMOG": [["BEMOG", "A210-"]]
+            },
+            "body": [],
+            "rwy": {
+                "07": [["ALIDO", "A160-"], ["DUNUM", "A130-"], "GABEV", ["KEMLU", "A50|S210-"], "SENNZ", ["MODON", "A30"], "VINRA", "#251"],
+                "14": [["ALIDO", "A100-"], "DUNUM", ["RAFNO", "A30"]],
+                "25": [["DUNUM", "A160-"], "MERKO", ["REVAL", "A50|S210"], ["XOLTI", "A30"], "IGTAD", "#072"],
```

Comment:
Missing ALIDO
DUNUM should be `["DUNUM", "A130-"]`
REVAL should be `["REVAL", "A50|S210-"]`
XOLTI should be `["XOLTI", "A30+"]`